app/packs/stylesheets/decidim/decidim_awesome/vote/vote.scss

```
@@ -0,0 +1,93 @@
+$rectangle-width: 7rem;
```

```suggestion
.voting-three-flag {
  $rectangle-width: 7rem;
```

we need to namespace all classes here, we need to take into account that this is only one implementation of the possibles.